antora/docs/modules/ROOT/pages/trusting_bundles.adoc

```diff
@@ -91,3 +91,18 @@ definitions used by RHTAP. The CI/CD process on that repository performs the fol
    includes the new list of acceptable bundles.
 
 The new Tekton Bundles become acceptable bundles as soon as the pull request merges.
+
+== Adding a new acceptable bundle to the Red Hat Trusted Application Pipeline.
+
+The process to add a new task defintion to the existing Red Hat acceptable task list is relatively uncomplicated and is detailed below.
+
+Before adding a new task, however, there are a few things to consider:
+
+* The added task should have a wide audience for usage. Tasks added to the build definitions repository should not be specific to a single product or team.
+* Submitting a task to the build-definitions repository has the practical effect of marking a task as "trusted" by Red Hat.
```

Comment:
Maybe one extra bullet point about who to talk to. "To discuss your ideas for a new task definition, visit <some slack channel>, or send an email to <??>.

Reply:
added.